app/models/Item.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace lms.models
+{
+    public abstract class Item
+    {
+        // Field List For Commands
+        public enum Field
+        {
+            // WorkItem
+            Title,
+            DueAt,
+            // Progress & Blockers
+            Description,
+            WorkItem // WorkItem(Id)
+        }
+
+        // Fields
+        public int Id { get; set; }
+
+        public DateTime CreatedAt { get; } = DateTime.Now;
+    }
+}
```

Comment:
I really like this idea for an item, but it is out-of-scope so I am going to ask you to revert this.

Generally, we want to make sure our branches are focused on the work being asked. Otherwise, this leads to scope creep. It also increases the risk of error. I don't mean to ramble here, but do feel free to chat with me on Slack if you want to discuss how this can affect risk.

I had an issue at work the other day where a small (2-line) but unrelated change was added by a dev which casued some regressions.